__tests__/app.test.js

```diff
@@ -111,4 +112,52 @@ describe("app", () => {
 
     })
 
+    describe("POST /api/articles/:article_id/comments", () => {
+        test("should return expected user and comment", () => {
+            return request(app).post("/api/articles/2/comments").send({
+                username: 'icellusedkars',
+                body: 'such a big fan wow'
+            }).expect(201).then((res) => {
+
+                const comment = res.body.comment
+
+                expect(comment.body).toBe("such a big fan wow");
+                expect(comment.author).toBe("icellusedkars");
+                expect(comment.article_id).toBe(2);
+                expect(comment.votes).toBe(0);
+
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
+                });
+            })
+        })
+
```

Comment:
It is possible for our client to include some unnecessary properties on the post body. Here we could use an additional 201 - ignores unnecessary properties